site/labs/objects/objects.rst

```diff
@@ -2,194 +2,91 @@
-
-
-.. code-block:: python
-    :linenos:	
-	
-    # Only need these if using multiple files
-    #from Person import *
-    #from ContactList import *
-
-    friends_list = ContactList()
-    friends_list.add_friend('Semaj', 'Sehguh', 'ssehguh@xfts.ca')
-    friends_list.add_friend('Greg', 'Smith', 'gsmith@xfts.ca')
-
-    print(friends_list)
-    print(len(friends_list))
-    a_friend = friends_list[1]
-    print(a_friend)
-
-    # This just makes sure that a_friend is 
-    # pointing to a a Person object.
-    # If it is, nothing special happens
-    # If it's not, it will crash the program
-    assert isinstance(a_friend, Person)
-
-    print(a_friend.get_first_name())
-    print(a_friend.get_last_name())
-    print(a_friend.get_email())
-    a_friend.set_first_name('Not')
-    a_friend.set_last_name('A')
-    a_friend.set_email('Thing')
-
-    print(friends_list)
-	
-	
-Make sure it makes sense to you *why* when we print out ``friends_list`` we now wee an altered person. 	
-	
-
-Add Some Things
-===============
+#. Create a ``Rectangle`` class
 
-Now that you have everything working, go add some additional attributes to the ``Person`` class, update methods, add new methods, use the methods in the ``ContactList`` class, etc. Basically I just want you to go nuts and see what you can do. 
+    * Attribute for ``length``
+    * Another attribute for ``width``
+    * The constructor will take two parameters for specifying the size
+    * Method for calculating ``area()``
+    * Method for calculating the ``perimeter()``
+    * Make instances of the class and write ``assert`` tests to verify correctness
 
-Maybe go even loop up other special python methods and see if you can hac them to work in weird ways. 
 
-Kattis Problems
-===============
+#. Create a ``RectangularPrism`` class
+    * Attributes for ``length``, ``width``, and ``height``
+    * The constructor will take three parameters for specifying the size
+    * A method for calculating ``surface_area()``
+    * A method for calculating ``volume()``
+    * Make instances of the class and write ``assert`` tests to verify correctness
 
-Can you do these with something other than lists? In fact, you might have to for the runtime requirements. You may come up with a perfect solution that will be correct 100% of the time; however, a correct solution is not necessarily a *good* solution. 
 
-1. https://open.kattis.com/problems/everywhere 
-2. https://open.kattis.com/problems/babelfish
-3. https://open.kattis.com/problems/oddmanout
-4. https://open.kattis.com/problems/securedoors
-5. https://open.kattis.com/problems/modulo
+#. Create a ``Person`` class
 
-LeetCode Problems
-=================
+    * Attribute for ``first_name``
+    * Attribute for ``last_name``
+    * Attribute for ``email_address``
+    * The constructor will take three parameters for specifying the values of the attributes
+    * A method returning a string of their ``full_name()``
```

Comment:
There's lots of definitions of _full name_. Maybe worth disambiguating here and saying that any combination of their first/last names is sufficient?